openbb_platform/providers/fmp/tests/test_fmp_fetchers.py

```diff
@@ -762,3 +763,16 @@ def test_fmp_historical_market_cap_fetcher(credentials=test_credentials):
     fetcher = FmpHistoricalMarketCapFetcher()
     result = fetcher.test(params, credentials)
     assert result is None
+
+
+@pytest.mark.record_http
+def test_fmp_form_13f_fetcher(credentials=test_credentials):
```

Comment:
In, `openbb_platform/extensions/equity/integration`, there are two test files for the integration tests, API and Python. A full set of parameters (can be None if the param is optional) with `provider='fmp'` needs to be added to the params of the endpoint test. Insert them below the "sec" provider params.